services/movie_session.py

```diff
@@ -42,3 +41,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
Code Style: Use one style of quotes in your code. Double quotes are preferable. Consider changing the single quotes in the list type hint to double quotes for consistency.